admin_guide/pruning_resources.adoc

```diff
@@ -287,9 +305,41 @@ $ oadm prune images --keep-tag-revisions=3 --keep-younger-than=60m --confirm
 $ oadm prune images --prune-over-size-limit --confirm
 ----
 
+[[pruning-images-secure-or-insecure]]
+=== Using secure or insecure connection
+
+The secure connection is preferred and recomended approach. It is done over
+HTTPS protocol with a mandatory certificate verification. The `prune` command
+always attempts to use it if possible. If not possible, in some cases it can
+fall-back to insecure connection which is dangerous. In this case, either
+certificate verification is skipped or plain HTTP protocol is used.
+
+The fall-back to insecure connection is allowed in the following cases unless
+`--certificate-authority` is specified:
+
+1. *(Dangerous)* The `prune` command is run with `--force-insecure` option.
+2. Provided registry-url is prefixed with `http://` scheme
+3. Provided registry-url is a local-link address or localhost.
+4. Config of the current user allows for insecure connection. This may be
+caused by the user either logging in using `--insecure-skip-tls-verify` or
+choosing the insecure connection when prompted.
+
+[IMPORTANT]
+====
+If the registry is secured by a certificate authority different from the one
+used by {product-title}, it needs to be specified using
+`--certificate-authority` flag. Otherwise, the `prune` command will fail with
+an error similar to
+xref:using-wrong-certificate-authority[the one] or
```

Comment:
similar to those seen in xref:using-wrong-certificate-authority[Using the Wrong Certificate Authority] or xref:using-insecure-connection-against-secured-registry[Using an Insecure Connection Against a Secured Registry].